### ISSUE TITLE
[4.x] `z-index` and positioning issues with Bard/Replicator new set popups

### ISSUE THREAD

**Issue by duncanmcclean** [statamic/cms#7704]
### Bug description

In testing out v4, I've noticed two bugs related to the `z-index` and positioning of the 'new set popup' for Bard & Replicator fields.

### Bard: Nav is positioned on top of the popup

<img width="718" alt="image" src="https://user-images.githubusercontent.com/19637309/225236740-310fd7b4-ff98-4ce8-bb89-f88917e76463.png">

As you can screen from the screenshot, the border from the nav is showing on top of the popup. Also, to the left of that border, you can't actually click on anything in the popup.

When I scroll down to the 'Updates' part in the nav, it's on top of the popup contents.

### Replicator: Popover no where near the 'Add Set' button

<img width="1026" alt="image" src="https://user-images.githubusercontent.com/19637309/225237214-d18dd15c-e1b0-4af5-971d-8c857d8dab8c.png">

With the Replicator, the popup is showing at the bottom of the screen and I can only see one of the options as the rest are off the screen.

### How to reproduce

1. Setup a v4 site, based on Peak
2. Clone down the `master` branch
3. Run `npm run dev`
4. Try to use Replicator & Bard fields

### Environment

```yaml
Environment
Application Name: Statamic 4 Test
Laravel Version: 9.52.4
PHP Version: 8.2.3
Composer Version: 2.2.6
Environment: local
Debug Mode: ENABLED
URL: statamic-4.test
Maintenance Mode: OFF

Cache
Config: NOT CACHED
Events: NOT CACHED
Routes: NOT CACHED
Views: CACHED

Drivers
Broadcasting: log
Cache: statamic
Database: mysql
Logs: stack / single
Mail: smtp
Queue: sync
Session: file

Statamic
Addons: 4
Antlers: runtime
Stache Watcher: Enabled
Static Caching: Disabled
Version: dev-master PRO

Statamic Addons
studio1902/statamic-peak-browser-appearance: 1.0
studio1902/statamic-peak-commands: 1.24
studio1902/statamic-peak-seo: 1.6.1
studio1902/statamic-peak-tools: 2.1
```

### Installation

Starter Kit using via CLI

### Antlers Parser

None

**Comment by robdekort** [comment]
I noticed in Safari it does it differently as well. Worth a good test probably.

**Comment by jasonvarga** [comment]
The set picker is completely changed in #7669 and makes this a non issue.

**Comment by robdekort** [comment]
I noticed these issues after #7669 got merged though. Re-tested just now. But maybe we're just too eager and early :-D.

**Comment by jasonvarga** [comment]
It's not merged into master.

**Comment by robdekort** [comment]
Gotcha!
